src/Parser.php

```diff
@@ -2681,7 +2699,7 @@ protected function value(&$out)
 
         // unicode range with wildcards
         if (
-            $this->literal('U+', 2) &&
+            $this->literal(strtolower('U+'), 2) &&
```

Comment:
why not writing it directly in lowercase ?

Reply:
I can do that, if you prefer. 

my reasoning was, that the actual notation in SCSS will have a uppercase U and like that we can easier find it in the codebase